helpers/log_context.py

```diff
@@ -124,15 +125,25 @@ def populate_from_sqlalchemy(self, dbsession):
 
         self._populated_from_db = True
 
+    def add_to_log_record(self, log_record: dict):
+        log_record["context"] = self.as_dict()
+
+    def add_to_sentry(self):
+        d = self.as_dict()
+        d.pop("sentry_trace_id", None)
+        sentry_sdk.set_tags(d)
```

Comment:
Interesting, I see [`set_tags`](https://getsentry.github.io/sentry-python/apidocs.html#sentry_sdk.Scope.set_tags) as a `Scope` API, but not as part of the [top level APIs](https://getsentry.github.io/sentry-python/api.html#sentry_sdk.api.set_tag).

Maybe this is just an oversight in the docs? CC @antonpirker